Cleanup the getting started guide (#175)
kudobuilder/www: content/docs/developing-operators/getting-started.md

# Getting Started

## Prerequisites

> It's time to create your first operator. But before you proceed make sure you're up-to-date on all the important KUDO concepts:

Reviewer: ```suggestion
It's time to create your first operator. But before you proceed, make sure you're up-to-date on all the important KUDO concepts:
```

> * [Operator packages](developing-operators/packages.md)

Author: ```suggestion
* [Operator packages](packages.md)
```
Links are relative to the current file. Here and below.

* [Plans](developing-operators/plans.md)
* [Tasks](developing-operators/tasks.md)
* [Parameters](developing-operators/parameters.md)
* [Templates](developing-operators/templates.md)

## Package Structure

> Out first operator will deploy nginx to the cluster. If you're thinkg that one doesn't need a KUDO operator to do this, you're absolutely right. It hardly showcases KUDOs stong sides but this is a `hello-world` type of operator and we have to start somewhere.

Reviewer: ```suggestion
Our first operator will deploy Nginx to the cluster. If you're thinking that one doesn't need a KUDO operator to do this, you're absolutely right. It hardly showcases KUDOs strong sides but this is a `hello-world` type of operator and we have to start somewhere.
```

> <<< @/kudo/test/integration/first-operator-test/first-operator/params.yaml

Reviewer: I think this section needs some more love. I agree that we should 't repeat ourselves, however:
- we jump directly into the yaml files, without even explaining *what operator we're building* and what should be expected at the end
- the _"Details for the concepts"_ part is important but should probably be placed at the beginning as a "Prerequisite" to the rest of it

> Now your first operator is ready and you can install it to your cluster. You can do this by invoking `kubectl kudo install ./first-operator` where `./first-operator` is a relative path to the folder containing your operator. To do this, you need to have the KUDO CLI installed - [follow the instructions here](cli.md), if you haven't already. Various resources will be installed for your operator, among them `Operator`, `OperatorVersion` and `Instance` as described in [What is KUDO](TODO).

Reviewer: ```suggestion
Now your first operator is ready and you can install it to your cluster. You can do this by invoking `kubectl kudo install ./first-operator` where `./first-operator` is a relative path to the folder containing your operator. To do this, you need to have the KUDO CLI installed - [follow the instructions here](cli.md), if you haven't already. Various resources will be installed for your operator, among them `Operator`, `OperatorVersion` and `Instance`, as described in, [What is KUDO](what-is-kudo.md).
```

<<< @/kudo/test/integration/first-operator-test/first-operator/params.yaml

> Now your first operator is ready and you can install it to your cluster. You can do this by invoking `kubectl kudo install ./first-operator` where `./first-operator` is a relative path to the folder containing your operator. To do this, you need to have the KUDO CLI installed - [follow the instructions here](cli.md), if you haven't already. Various resources will be installed for your operator, among them `Operator`, `OperatorVersion` and `Instance`, as described in, [What is KUDO](what-is-kudo.md).

Author: ```suggestion
Now your first operator is ready and you can install it to your cluster. You can do this by invoking `kubectl kudo install ./first-operator` where `./first-operator` is a relative path to the folder containing your operator. To do this, you need to have the KUDO CLI installed - [follow the instructions here](cli.md), if you haven't already. Various resources will be installed for your operator, among them `Operator`, `OperatorVersion` and `Instance`, as described in, [What is KUDO](../what-is-kudo.md).
```
